src/Conversion/ONNXToKrnl/Math/Softmax.cpp

```diff
@@ -273,17 +276,10 @@ struct ONNXSoftmaxOpLowering : public ConversionPattern {
     Value negInfinity = create.math.constant(
         elementType, -std::numeric_limits<float>::infinity());
 
-    if (opset < 13)
-      // For Softmax opset < 13, `axis` is the coerced point. All dimensions
-      // after `axis` will be logically coerced into a single dimension.
-      emitInstForSoftmaxBeforeV13(
-          rewriter, loc, alloc, input, sumOp, maxOp, zero, negInfinity, axis);
-    else
-      // For Softmax opset 13, `axis` attribute indicates the dimension along
-      // which Softmax will be performed. No need to coerce the dimensions after
-      // `axis`.
-      emitInstForSoftmaxV13(
-          rewriter, loc, alloc, input, sumOp, maxOp, zero, negInfinity, axis);
+    // For Softmax opset < 13, `axis` is the coerced point. All dimensions
+    // after `axis` will be logically coerced into a single dimension.
```

Comment:
The comment seems out of place here. I think the comments of the if-then-else should migrate to the relevant `emitInstForSoftmax<>` specialization.